app/routes/index.js

```diff
@@ -2,31 +2,28 @@ import Ember from 'ember';
 import config from 'ember-get-config';
 import ResetScrollMixin from '../mixins/reset-scroll';
 
+const getTotalPayload = '{"size": 0, "from": 0,"query": {"bool": {"must": {"query_string": {"query": "*"}}, "filter": [{"term": {"type.raw": "preprint"}}]}}}';
+
 export default Ember.Route.extend(ResetScrollMixin, {
     model() {
-        var getTotalPayload = '{"size": 0, "from": 0,"query": {"bool": {"must": {"query_string": {"query": "*"}}, "filter": [{"term": {"type.raw": "preprint"}}]}}}';
-
-        var sharePreprintsTotal = Ember.$.ajax({
-                type: 'POST',
-                url: config.SHARE.searchUrl,
-                data: getTotalPayload,
-                contentType: 'application/json',
-                crossDomain: true,
-            }).then(function (results) {
-                return results.hits.total.toLocaleString();
-            });
+        return this.store.query('taxonomy', { filter: { parents: 'null' }, page: { size: 20 } });
+    },
+    setupController(controller, model) {
+        this._super(controller, model);
+        controller.set('currentDate', new Date());
 
-        return Ember.RSVP.hash({
-            theDate: new Date(),
-            subjects: this.store.query('taxonomy', { filter: { parents: 'null' }, page: { size: 20 } }),
-            sharePreprintsTotal: sharePreprintsTotal
-        });
+        // Fetch total number of preprints. Allow elasticsearch failure to pass silently.
+        Ember.$.ajax({
```

Comment:
It sounds like this comes down to a question of how often we expect the SHARE count to increment. Since you're more familiar with that infrastructure, I've gone ahead and pushed the requested changes.